src/ruby/yast/core_ext/string/ansi.rb

```diff
@@ -0,0 +1,18 @@
+class String
+  # Removes color and cursor escape codes (modifying the string)
+  # http://en.wikipedia.org/wiki/ANSI_escape_code
+  def remove_ansi_sequences
+    remove_ansi_color_sequences
+    remove_ansi_cursor_sequences
+  end
+
+  # Removes ANSI color escape codes (modifying the string)
+  def remove_ansi_color_sequences
```

Comment:
`but maybe sometimes` => over-engineering
I propose to have it private and if in future such demand appear, then make it public with proper version dump. Otherwise we may find in future, that we need something slightly different then just `remove_ansi_color_sequences`